invidious/umbrel-app.yml

```diff
@@ -2,7 +2,7 @@ manifestVersion: 1
 id: invidious
 category: Social
 name: Invidious
-version: "2022.09.08-53fb6ad"
+version: "2022.12.02-5160d8b"
```

Comment:
Is this version correct? When I load the app, it's showing: `2022.11.22-5160d8ba`

Reply:
fixed